src/pages/admin/SelfAffirmation.jsx

```diff
@@ -1,76 +1,115 @@
-import React from 'react';
-import Button from '../../components/common.jsx/button';
+import React from "react";
+import Button from "../../components/common.jsx/button";
 
 const SelfAffirmation = () => {
-    return ( 
-        <div className='px-4 bg-white'>
```

Comment:
why did you make changes to the admin selfAffirmation page ? If you want to lift things from there you can do that but don't make changes to what you're not working on

Reply:
It was when I was having issues trying to render the page the first time, but it's resolved now.